prow/cluster/crier_rbac.yaml

```diff
@@ -33,6 +33,14 @@ rules:
     - "watch"
     - "list"
     - "patch"
+- apiGroups:
+    - ""
+  resources:
+    - "pods"
+    - "events"
+  verbs:
+    - "get"
+    - "list"
```

Comment:
do we even have RBAC in prow.k8s.io in either cluster?

Reply:
No (which is why it works), but these files are the basis for the rbac setup used in other clusters.